week3/homework/src/index.js

```diff
@@ -1,3 +1,50 @@
 'use strict';
 
-// TODO: Write the homework code in this file
+const express = require('express');
+const app = express();
+const {
+  createTodo,
+  updateTodo,
+  readTodos,
+  deleteTodo,
+  readTodo,
+  clearTodos,
+  markAsDone,
+  markAsNotDone
+} = require('./actions');
+
+// app.post('/todos', (req, res) {
+//   createTodo();
+// })
+app.post('/todos', (req, res) => {
+  createTodo(req, res);
+});
+
+app.get('/todos', (req, res) => {
+  readTodos(req, res)
+    .then(todos => res.status(200).json(todos))
+    .catch(error => res.status(500).send("can't read file!", error));
+});
+
+app.put('/todos/:id', (req, res) => {
```

Comment:
very pleasant to read

Reply:
Thanks a lot for your feedback! It's very appreciated! :)